<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary</class>
 <widget class="QMainWindow" name="Summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Summary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbLogin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddData">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbViewData">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>View Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lePassword</tabstop>
  <tabstop>pbLogin</tabstop>
  <tabstop>pbAddData</tabstop>
  <tabstop>pbViewData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
